docker-compose.yml

```diff
@@ -7,11 +7,13 @@ server:
     - "8080:8080"
   volumes:
     - ~/.dockercfg:/root/.dockercfg
+    - /var/lib/boot2docker/tls:/root/certs
```

Comment:
This seems to also work for me:

```
- "$DOCKER_CERT_PATH:/root/certs:ro"
```

Which means it'll work without the boot2docker dependency.